lerobot/common/datasets/push_dataset_to_hub/_convert_datasets.py

```diff
@@ -0,0 +1,112 @@
+import argparse
+from pathlib import Path
+
+from lerobot import available_datasets
+from lerobot.common.datasets.push_dataset_to_hub._download_raw import download_raw
+from lerobot.scripts.push_dataset_to_hub import push_dataset_to_hub
+
+SOURCE_FORMAT = {
+    "aloha": "aloha_hdf5",
+    "pusht": "pusht_zarr",
+    "xarm": "xarm_pkl",
+    "umi": "umi_zarr",
+}
+
+
+def convert_datasets(
+    local_dir: Path,
+    raw_repo_ids: list[str],
+    push_repo: str,
+    vcodec: str,
+    pix_fmt: str,
+    g: int,
+    crf: int,
+    dry_run: bool = False,
+) -> None:
+    if len(raw_repo_ids) == 1 and raw_repo_ids[0].lower() == "all":
+        raw_repo_ids = [f"cadene/{id_.split('/')[1]}_raw" for id_ in available_datasets]
```

Comment:
Make it configurable maybe?
```suggestion
        raw_repo_ids = [f"{hf_user}/{id_.split('/')[1]}_raw" for id_ in available_datasets]
```

Reply:
UPDATE:
```diff
parser.add_argument(
        "--raw-repo-ids",
        type=str,
        nargs="*",
        default=["lerobot-raw"],
-       help="Dataset repo ids. if 'all', the list from `available_datasets` will be used.",
+       help="""Raw dataset repo ids. if 'lerobot-raw', the keys from `AVAILABLE_RAW_REPO_IDS` will be
+           used and raw datasets will be fetched from the 'lerobot-raw/' repo and pushed with their
+           associated format. It is assumed that each dataset is located at `raw_dir / raw_repo_id` """,
    )
```

Reply:
(I don't handle downloading raw datasets in this script anymore as it should be done outside)